drivers/modem/ublox-sara-r4.c

```diff
@@ -1574,8 +1583,15 @@ static int offload_connect(void *obj, const struct sockaddr *addr,
 		return 0;
 	}
 
+	ret = modem_context_sprint_ip_addr(addr, ip_str, sizeof(ip_str));
+	if (ret != 0) {
+		errno = -ret;
+		LOG_ERR("Error formatting IP string %d", ret);
+		return -1;
+	}
+
 	snprintk(buf, sizeof(buf), "AT+USOCO=%d,\"%s\",%d", sock->id,
-		 modem_context_sprint_ip_addr(addr), dst_port);
+			ip_str, dst_port);
```

Comment:
same here

Reply:
Fixed, I also remove the spurious space

Reply:
Thanks. The space is not spurious though -- without the space, it is no longer properly aligned:
```
	snprintk(buf, sizeof(buf), "AT+USOCO=%d,\"%s\",%d", sock->id,
		 modem_context_sprint_ip_addr(addr), dst_port);
		ip_str, dst_port);
```

Reply:
Looks to be resolved, based on the Coding Guidelines checks.